src/ELZ-0-2-0-beta/Test/Receiving-XIS/elz-receiving-xis-scenario-1-maximal.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="client">
+    <id value="elz-receiving-xis-scenario-1-maximal"/>
+    <version value="R4"/>
+    <name value="ELZ - Receiving XIS - Scenario 1 - Retrieve ELZ resources of van-XXX-Sanderink-de-Kok"/>
+    <description value="Scenario 1 - Retrieve ELZ resources of van-XXX-Sanderink-de-Kok"/>
+    <nts:authToken patientResourceId="patient-van-XXX-Sanderink-de-Kok"/>
+
+    <test id="01-retrieve-Task">
+        <name value="Scenario 1.1 - Task"/>
+        <description
+            value="Query Task resource"/>
+        <nts:include value="test.client.search" scope="common"
+            resource="Task"
+            params="?_id=task-hg-VerwijzingHuisartsParamedicusMaximaalbericht"
+            useToken="true"/>
+        <nts:include value="canary-assert.response.successfulSearch" scope="common"/>
```

Comment:
Kloppen deze asserts dan nog wel voor een Read ipv een Search (oprecht geen idee en ook niet uitgezocht)

Reply:
Dit is dan inderdaad niet meer nodig door de aanpassing, dus ik heb de assert eruit gehaald.